packages/rolldown/tests/fixture.test.ts

```diff
@@ -22,8 +22,14 @@ function main() {
         const output = await compileFixture(
           nodePath.join(import.meta.dirname, dirPath),
           testConfig,
-        )
-        if (testConfig.afterTest) {
+        ).catch(async (err) => {
+          if (testConfig.catchError) {
+            await testConfig.catchError(err)
+            return
+          }
+          throw err
+        })
```

Comment:
An `async` function is passed to catch because it allows to throw new errors in TestConfig#catchError